test/utils_test.js

```diff
@@ -35,9 +35,7 @@ suite('docker_utils', function() {
         docker.getImage(image).inspect().then(function image(result) {
           assert.ok(result);
           done();
-        }).catch(
-          done
-        );
+        }, done);
```

Comment:
This changes semantics... because `assert.ok(result)` errors won't be catched...